sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -479,6 +485,7 @@ def initialize
       self.traces_sampler = nil
       self.enable_tracing = nil
       self.enable_logs = false
+      self.send_stdlib_logs = false
```

Comment:
This is not needed, there's a config option called `enabled_patches` that should be used instead. The Logger patch should be registered as `:logger` and then the users will be able to configure it to be enabled by default.

You can see how `:http` patch is done for reference.